vtex-apps/admin-pages
# Page level authentication

#### What is the purpose of this pull request?
This pull request removes the **View** button from Page Lists and allows user to set the authentication for each page.

#### What problem is this solving?
Now admins can set the authentication for each page.

#### How should this be manually tested?
``vtex link`` this app. You can check the toggle button "Requires login?" at **Edit Page** page.

#### Screenshots or example usage

#### Types of changes
